charts/core/src/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,88 @@
+import { useEffect } from 'react';
+
+import { useDarkMode } from '@leafygreen-ui/leafygreen-provider';
+import {
+  borderRadius,
+  color,
+  InteractionState,
+  spacing,
+  Variant,
+} from '@leafygreen-ui/tokens';
+
+import { useChartContext } from '../ChartContext';
+
+import {
+  SortDirection,
+  SortKey,
+  SortOrder,
+  TooltipProps,
+} from './Tooltip.types';
+
+export function Tooltip({
+  sortDirection = SortDirection.Desc,
+  sortKey = SortKey.Value,
+  valueFormatter = value => `${value} GB`,
```

Comment:
The thing I didn't like about just showing the value was that it doesn't demonstrate that there's a way to format the value. Since it's a function, I didn't think it would be a terribly useful storybook option. Do we typically expose functions in some way as an option in storybook?

For context, being able to do this formatting has been a big ask from stakeholders. My guess is that almost everyone who uses this component, at least at first, will be formatting their values in some way. That's why I wanted to make sure it was demonstrated.

Reply:
One more piece of context is that this is also being done on the axis. Open to not doing this if we normally don't or there's a better way to express this!

Reply:
> marking the prop required make more sense to me from a consumer-perspective
I don't think it should be required. There are cases where a value might just be a value, and I think forcing consumers to always format their value shouldn't be necessary.

> I think having "GB" as the default unit of measure is overly specific
I totally get this. I guess the fear is that the "GB" isn't part of the format but rather that it looks like the chart always renders with "GB". 

The workarounds still seem a bit misleading since as you noted, it seems to indicate that you can only add before or after the value. I say we just remove it as you first suggested. The docs will clearly show the formatter prop, so hopefully it shouldn't be necessary in the visualization. I'll remove it from the axis as well and push the change shortly!

Reply:
@stephl3! I apologize! I thought this was in storybook but I had accidentally left it as a default prop. You probably thought I was losing my mind lol. Totally right to call this out. 

Question though, should we default to a format in the storybook story? Or do you think we should leave the formatting off completely?

Reply:
I think I'm going to remove it from there too. You incidentally convinced me when I thought that's what you were referring to 😅. Any issue with that though, please let me know!